### DISCUSSION TITLE
[xy-chart] How to make some reference always visible in Y axis?

### DISCUSSION THREAD

**Discussion by RIP21** [airbnb/visx#1003] [Q&A] [answered]
Hard to explain but here is the case
![image](https://user-images.githubusercontent.com/3940079/103948394-fe844880-5139-11eb-953d-81a991a4af9e.png)
 I have the same value in the dataset, but different dates. And it's just a line. How to make it into at least 3 points of data below or above? So it looks like, at least half-filled with something and not just plain line. Or maybe just it should fill the area down in that scenario by default? Code of the graph is mentioned here: https://github.com/airbnb/visx/discussions/1000

**Comment by williaster** [comment, marked as answer]
Hey @RIP21, if I understand correctly, you want the # points to be evident? One option would be to render both a `LineSeries` **and** a `GlyphSeries` on top of each other with the same `data` (even the same `dataKey` should work). That would result in something like this (from the https://airbnb.io/visx/xychart demo, turning on glyphs + lines)

<img src="https://user-images.githubusercontent.com/4496521/103967390-c4557f80-5116-11eb-8261-7a85a4cc0a60.png" width="500" />

**Reply by RIP21** [reply]
No, I meant that I have AreaSeries but in case I have on Y axis always the same value, there is basically no area, just line :) 
![image](https://user-images.githubusercontent.com/3940079/104039951-545afe00-51d7-11eb-968c-c0c0cfcee56a.png)
![image](https://user-images.githubusercontent.com/3940079/104039962-59b84880-51d7-11eb-9b18-45e60b937724.png)

I would want to have this area gradient goes to X axis on the first example but it just a line :)

**Reply by williaster** [reply]
Ohhh I see what you mean, I think this is a bug actually. Will get a fix out shortly (looks like you can't convert discussions to issues yet).

**Reply by williaster** [reply]
Have a fix out here #1008

**Reply by RIP21** [reply]
Nice! Can't wait for merge and publish :)
